lars-reimann/api-editor
# feat: Calling rest endpoint "infer"

Closes #282

### Summary of Changes
- Added a button in the frontend to send the API data and annotations to the backend over the REST interface
- Created an inferable data model in the frontend
- Transform package data and annotations into the inferable data model
- Send a POST request to the backend containing the inferable data model as JSON and receive the response

### Testing instructions

- Check whether the "Infer"-button correctly sends a request to the backend.
- Check whether the backend correctly receives the annotated python package data
